Hardware/index.md

```diff
@@ -14,7 +14,9 @@ seL4 is formally verified for specific configurations for a subset of these
 platforms. The depth of the proofs and which properties are verified depend on
 the platform.
 
-The tables below provide more details.
+The tables below provide more details on [maintained](#maintained-platforms)
+platforms. You can also find below a table of
+[unmaintained](#unmaintained-platforms) platforms if needed.
```

Comment:
I'm not sure we should mention the unmaintained platforms up here. It's good to list them at the bottom for completeness so people know that maybe finishing or reviving a port is faster than starting one from scratch, but these are basically platforms that are not really usable, and the page is for finding out what I can use.

Reply:
reverted